data/input_files/tests/nonlinear_solid/serac_dynamic_solver/dyn_linesearch_solve.lua

```diff
@@ -0,0 +1,66 @@
+-- Comparison information
+expected_x_l2norm = 12.86733
+expected_v_l2norm = 0.22298
+epsilon = 0.0001
+
+-- Simulation time parameters
+dt      = 3.0
+t_final = 6.0
+
+main_mesh = {
+    -- mesh file
+    mesh = "../../../../meshes/beam-hex.mesh",
+    -- serial and parallel refinement levels
+    ser_ref_levels = 1,
+    par_ref_levels = 0,
+}
+
+-- Solver parameters
+nonlinear_solid = {
+    solver = {
+        linear = {
+            rel_tol     = 1.0e-4,
+            abs_tol     = 1.0e-8,
+            max_iter    = 500,
+            print_level = 0,
+            solver_type = "gmres",
+            prec_type   = "AMG",
+        },
+
+        nonlinear = {
+            rel_tol     = 1.0e-4,
+            abs_tol     = 1.0e-8,
+            max_iter    = 500,
+            print_level = 1,
+            solver_type = "KINLineSearch",
+        },
+
+        dynamic = {
```

Comment:
I think these are the parameters for the mass matrix inversion. It might be clearer to name it `mass_solver` or something like that. The other parameters could be `stiffness_solver` for additional clarity.

Reply:
Essentially, the user should know that these parameters are for the mass matrix solve and the other parameters are for the stiffness solve.